### DISCUSSION TITLE
Transfer a license from Chrome to firefox?

### DISCUSSION THREAD

**Discussion by zoroasterthecat** [aclap-dev/video-downloadhelper#1520] [License & Purchase requests]
### What's your issue?

I purchased a license for Chrome, but would like to use it on Firefox, is it possible to migrate the license?

### The 10 first digits (only 10) of your license number or email address

7cebc39045

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
We give one month for unconditional refund, we can even make an effort sometimes, but your license was purchased in August.

**Reply by MathieuleBleu** [reply]
@jcfrog Hi, what does make an effort sometimes mean? I bought at the end of November and I didnt know the unconditional refund exists I checked the extension website then and there was nothing. I now found it mentioned here (https://github.com/aclap-dev/video-downloadhelper/wiki) after coming across the post here.
I know its also quite some time but I just assumed there was no refund back than and counted it as a loss. I would appreciate if at least the one sentence could be added to the faq. And if you make an exception to refund so I can use it in firefox Id appreciate that more. On chrome I cant download most videos I want to on chrome even with the free version it does work. Thats why I wanna change to firefox. Thank you

**Reply by jcfrog** [reply]
Sorry it's not possible.
jerome

**Reply by MathieuleBleu** [reply]
Thanks for coming back to me @jcfrog is it possible to get a 50% discount on the firefox version?

**Reply by jcfrog** [reply]
No, sorry, no such discount.
jerome
